### PR TITLE
CRAFTERCMS-1727

### PR DESCRIPTION
**Pull request by dejan-brkic**
Added studio id field and internal service

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/1727

### CODE REVIEW

**Review by sumerjabri — CHANGES_REQUESTED** [review]
please fix the Codacy issues that can be fixed.

File: src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy [hunk 1]
```diff
@@ -20,5 +20,7 @@ if(resource != null) {
 }
 else {
 	response.setStatus(404)
-	return "NOT FOUND"
+    def instance = applicationContext.get("instanceService")
```
Comment (sumerjabri) [inline src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy]: Is this related to the instance ID?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy (hunk 1 above)
Comment (sumerjabri) [reply src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy]: @dejan-brkic ^

Comment (dejan-brkic) [reply src/main/webapp/default-site/scripts/rest/api/1/server/get-ui-resource-override.get.groovy]: I have removed that with the new commit. It was a mistake.

**Review by sumerjabri — APPROVED** [review]
